install/kubernetes/cilium/values.yaml

```diff
@@ -1058,6 +1061,9 @@ prometheus:
     # This requires the prometheus CRDs to be available (see https://github.com/prometheus-operator/prometheus-operator/blob/master/example/prometheus-operator-crd/monitoring.coreos.com_servicemonitors.yaml)
     #
     enabled: false
+    # -- Labels to add to ServiceMonitor cilium-agent
+    #
```

Comment:
nit: I think we can remove the extra empty comment line, bonus points for removing the one at line 1062 as well.

```suggestion
```

Reply:
I see quite many extra empty comment lines in the values.yaml. Should they be removed all for consistency?

Reply:
I removed some extra comment lines, please check them out. Some others should be intact.